spring-ai-alibaba-agent-framework/src/main/java/com/alibaba/cloud/ai/graph/agent/node/AgentLlmNode.java

```diff
@@ -75,11 +76,15 @@ public class AgentLlmNode implements NodeActionWithConfig {
 
 	private ChatClient chatClient;
 
+	private ChatModel chatModel;
+
+	private ChatOptions chatOptions;
```

Comment:
ChatOptions is not used in any place? What's the purpose of adding this new filed?

Reply:
same as chatModels